x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/VerifierTests.java

```diff
@@ -2590,6 +2590,118 @@ private void checkFullTextFunctionNullArgs(String functionInvocation, String arg
         );
     }
 
+public void testGreatestLeastWithUnsupportedDataTypes() {
```

Comment:
This doesn't look correctly indented, which probably the CI will complain about. You should be able to format these files with:

```
./gradlew :x-pack:plugin:esql:spotlessApply 
```